<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>下拉列表</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="btnInitCb">
          <property name="text">
           <string>初始化列表</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>清除列表</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="ckEnableCbxEdit">
          <property name="text">
           <string>启用编辑</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QComboBox" name="comboBoxData">
          <item>
           <property name="text">
            <string>北京市</string>
           </property>
           <property name="icon">
            <iconset resource="icon.qrc">
             <normaloff>:/icons/images/3.ico</normaloff>:/icons/images/3.ico</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>上海市</string>
           </property>
           <property name="icon">
            <iconset resource="icon.qrc">
             <normaloff>:/icons/images/3.ico</normaloff>:/icons/images/3.ico</iconset>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>初始数据</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="btnInit">
          <property name="text">
           <string>初始化城市+区号</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="comboBoxInit"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnAppend">
        <property name="text">
         <string>追加内容到下拉列表</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnClearText">
        <property name="text">
         <string>清除文本内容</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="ckEnableText">
        <property name="text">
         <string>启用编辑</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
